css_docs/products.css

```diff
@@ -0,0 +1,129 @@
+.page-container {
+    position: relative;
+    display: grid;
+  color: var(--ecom-login-bdr);
+    grid-template-columns: repeat(5,1fr);
+    grid-gap: 10px;
+    grid-template-areas:
+      "head"
+      "side"
+      "info"
+      "content"
+      "footer";
+  
+    grid-template-areas:
+      " head head head head head"
+      " side info info info info"
+      " side content content  content content"
+      " footer footer footer footer footer";
+}
+
+
+
+.page-header {
+    grid-area: head;
+   position: sticky;
+    top: 0;
+    z-index: 1;
+  }
+  
+
+  
+  .side-bar {
+      
+    grid-area: side;
+    display: flex;
+    justify-content: flex-start;
+    position: sticky;
+      z-index: 0;
+     
+  }
+  .info{
+      grid-area: info;
+      display: flex;
+      align-items: center;
+    
+  }
+  
+  .page-content {
+    grid-area: content;
+   display: grid;
+grid-template-columns: repeat(4,1fr);
+gap:5px;
+align-items: flex-start;
+}
```

Comment:
Look once for indentation. use prettier for this if you are not using it already.